src/SFA.DAS.AssessorService.Api.Types/Models/AO/EpaOrganisationStandard.cs

```diff
@@ -9,7 +9,7 @@ public class EpaOrganisationStandard
         public string OrganisationId { get; set; }
         public int StandardCode { get; set; }
         public string StandardReference { get; set; }
-        public List<string> StandardVersions { get; set; }
+        public List<OrganisationStandardVersion> StandardVersions { get; set; }
```

Comment:
This has changed due to the request, I can't see the same change on the model Admin Service side, should there be? (Admin side screenshot)
![image](https://user-images.githubusercontent.com/54770494/143033393-5d75489f-2c5e-4c82-a1f9-a5f82aab16c4.png)